ferrocene/doc/internal-procedures/src/docs/signatures.rst

```diff
@@ -0,0 +1,73 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+Digital signatures
+==================
+
+All the qualification documents we send to TUV have to be digitally signed by
+the responsible parties, to attest they were reviewed and to prevent accidental
+changes to the documents (which would require TUV to review the documents
```

Comment:
```suggestion
All the qualification documents we send to the assessor have to be digitally signed by
the responsible parties, to attest they were reviewed and to prevent accidental
changes to the documents (which would require the assessor to review the documents
```